Makefile

```diff
@@ -57,10 +58,12 @@ isclean:
 .PHONY: build
 build: isclean envtest
 	docker build . -f build/ci-operator/Dockerfile -t $(OPERATOR_IMAGE_URI)
+	docker tag $(OPERATOR_IMAGE_URI) $(OPERATOR_IMAGE_URI_LATEST)
 
 .PHONY: push
-push: build
```

Comment:
I think we want to `build` before pushing to ensure the image locally is up to date?

Reply:
It's great in practice but not necessary.  If there were changes the image won't exist (it's based on git hash).  Also, the build is kind of slow.  I hit this because I messed up the push of the latest tag and noticed it was going to build again.  I didn't want to wait.  I always do a `make build push` explicitly anyway.  Not going to loose any sleep over putting `build` back in as a dep here tho if you feel strongly.

Reply:
I don't feel all that strongly about it. There's probably enough guardrails to keep us from shooting ourselves in the foot (that is, the default behaviour pushes to $USER registry).